app/exporters/odt/cv.rb

```diff
@@ -110,12 +110,12 @@ def project_description(project)
     end
 
     def formatted_year(obj)
-      if obj.year_to.nil?
-        "#{obj.year_from} - heute"
-      elsif obj.year_from == obj.year_to
-        obj.year_to
+      if obj.date_to.nil?
```

Comment:
is it possible to call these attrs: start_at, finish_at ?
would be more rails like